resources/searchview/main.js

```diff
@@ -99,13 +103,6 @@
 
         appState.query = query;
 
-        /** @type HTMLInputElement|null */
-        var badge = document.querySelector(".js-filter:checked");
-
-        if (badge) {
-            query += " -badge:" + badge.value;
-        }
-
```

Comment:
Since there is no more `upcoming` or `current` options, this was no longer necessary ( SFCC docs only lets you search current versions ).